committers.md

```diff
@@ -62,6 +62,7 @@ navigation:
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
 |Shixiong Zhu|Databricks|
+|Saisai Shao|Hortonworks|
```

Comment:
@jerryshao, it might be not a big deal but I remember I placed my name in alphabetical order of last name ..

Reply:
Ohh, I see, I will change it.